src/generator/operations.ts

```diff
@@ -587,6 +587,17 @@ function emitPagerDefinition(op: Operation, imports: ImportManager): string {
   return text;
 }
 
+function genApiVersionDoc(apiVersions?: ApiVersions): string {
+  if (!apiVersions) {
+    return '';
+  }
+  const versions = new Array<string>();
+  apiVersions.forEach((val) => {
+    versions.push(val.version);
+  })
+  return `// Uses API version ${versions.join(',')}\n`;
```

Comment:
Is there defensive coding or are there really instances multiple versions are the basis for code gen?

Reply:
I asked @timotheeguerin about when we'd have more than one API version for an operation.

Reply:
nit, "//  Generated from API version ${versions.join(',')}, See {hyperlink to swagger}\n"

Might be a lot of redundant text

Reply:
I don't know that we have the swagger information available.

I can change the text as specified above if you think that's better.